### PR TITLE
[TTAHUB-2510] Add IST visit dropdown to session form

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

Add drop down for IST visit and add audience of TTA/RO

## How to test

- Session form, participants section
- New field: "is this an IST visit?" 
- If no, you can choose recipients
- If yes, you choose from a list of roles

Confirm that the above works and matches the design. Note: this is deployed to sandbox currently.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2510

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/SessionForm/components/ParticipantsReadOnly.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/SessionForm/components/ParticipantsReadOnly.js:1]: Do we have a test for this that I haven't gotten to yet?

Comment (thewatermethod) [reply frontend/src/pages/SessionForm/components/ParticipantsReadOnly.js:1]: It's tested in participants.js. Most of this is just moved from that file as cleanup. I added an additional test to account for the new path of IST visit.

Comment (AdamAdHocTeam) [reply frontend/src/pages/SessionForm/components/ParticipantsReadOnly.js:1]: nvm

File: frontend/src/pages/SessionForm/pages/__tests__/participants.js [hunk 2]
```diff
@@ -253,14 +273,52 @@ describe('participants', () => {
...
+        event: {
+          pocIds: [userId],
+        },
+        recipients: [
+          {
+            id: 1,
+            label: 'R1 R1 G1',
+          },
+        ],
+        deliveryMethod: 'hybrid',
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/SessionForm/pages/__tests__/participants.js:291]: Do we need another test for the isHybrid branch?

File: frontend/src/pages/SessionForm/pages/participants.js [hunk 3]
```diff
@@ -21,57 +22,75 @@ import FormItem from '../../../components/FormItem';
...
 import useTrainingReportTemplateDeterminator from '../../../hooks/useTrainingReportTemplateDeterminator';
 import UserContext from '../../../UserContext';
-import PocCompleteView from '../../../components/PocCompleteView';
-import ReadOnlyField from '../../../components/ReadOnlyField';
 import RecipientsWithGroups from '../../../components/RecipientsWithGroups';
+import ParticipantsReadOnly from '../components/ParticipantsReadOnly';
 
 const placeholderText = '- Select -';
 
+const ROLE_OPTIONS = [
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/SessionForm/pages/participants.js:30]: Not sure if we care to stick these in common.

File: frontend/src/pages/SessionForm/pages/__tests__/participants.js (hunk 2 above)
Comment (thewatermethod) [reply frontend/src/pages/SessionForm/pages/__tests__/participants.js:291]: "Hybrid" wasn't added in this PR, but there's one in this file.

File: frontend/src/pages/SessionForm/pages/participants.js (hunk 3 above)
Comment (thewatermethod) [reply frontend/src/pages/SessionForm/pages/participants.js:30]: Only if we are going to use it somewhere else, I think

**Review by AdamAdHocTeam — COMMENTED** [review]
Not sure if this bug is linked to the changes but I don't have the ability to mark the event as complete:

image

**Pull request merged**